CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # PowerShell Editor Services Release History
 
+## v2.4.3
+### Wednesday, May 26, 2021
+
+-  [PowerShellEditorServices #1491](https://github.com/PowerShell/PowerShellEditorServices/pull/1491) - Bump OmniSharp to `v0.19.2`.
```

Comment:
No emoji here?

Reply:
Must not have labeled it.

Reply:
Weird, it should actually still end up with two emoji.